modules/images-samples-operator-deprecated-image-stream.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/configuring-samples-operator.adoc
+
+
+[id="images-samples-operator-deprecated-image-stream_{context}"]
+= Removing deprecated image stream tags from the Cluster Samples Operator
+
+The Cluster Samples Operator leaves deprecated image stream tags in an image stream because users can have deployments that use the deprecated image stream tags.
+
+You can remove deprecated image stream tags by editing the image stream with the  `oc tag` command.
+
+[NOTE]
+====
+Deprecated image stream tags that the samples providers have removed from their image streams are not included on initial installations.
+====
+
+.Prerequisites
+
+* You installed the `oc` CLI.
+
+.Procedure
+
+* Remove deprecated image stream tags by editing the image stream with the  `oc tag` command.
++
+[source,terminal]
+----
+$ oc tag -d <image-name:tag>
```

Comment:
s/image-name/image_stream_name/g

The following is same. To keep consistent with installation-images-samples-disconnected-mirroring-assist.adoc  module

Reply:
Thank you!